content/academy/apify_platform/publishing-your-actor.md

```diff
@@ -0,0 +1,68 @@
+﻿---
+title: Publishing your actor
+description: Prepare your actor for Apify Store with a description and README file and learn how to make your actor available to the public.
+menuWeight: 1
+category: courses
+paths:
+  - publishing-your-actor
+---
+
+# [](#publishing-actor) Publishing your actor
+
+Before you make your actor public, it is important to write a good **Description** and **README** sections, so that users can understand what your actor does, how to configure its inputs, and what kind of output it returns.
+
+You can find the Title and description configurations by going to **Apify Console** → **Actors** → **My Actors** → **your-actor** → **Settings → Publication**
+
+![Actor publication settings](https://i.postimg.cc/hvQf0M1B/Untitled.png)
+
+By optimizing your actor’s description and README for SEO, you can significantly increase the traffic of users to your actor’s page and, consequently, improve your chances of getting more users.
+
+We know this can be a daunting task to take on your own, so we prepared an in-depth [SEO guide](https://docs.apify.com/actors/publishing/seo-and-promotion) to help you **fine-tune your actor’s title, description, and Readme.**
+
+Now, let’s move on to understand exactly how to add a description and README to your Apify actors.
+
+## [](#description) Description
+
+The actor’s description is a short paragraph describing the actor’s purpose. It will be displayed on the actor’s page right below its title.
+
+![Actor title and description](https://i.postimg.cc/NFbBmHWc/actor-title-description.png)
+
+While writing your actor’s description you also have the option to write an SEO title and description. The SEO title and description are used in place of the actor name and description on search engine results pages. Good SEO titles and descriptions utilize popular keywords, summarize the actor functionality, and are between 40-50 and 140-156 characters long, respectively.
+
+![SEO title and description](https://i.postimg.cc/HL0KrgGG/actor-SEO.png)
+
+## [](#readme) README
+
+The next step is to write a **README** detailing the actor’s features, reasons to scrape the targeted website, and an explanation about how to use the actor.
```

Comment:
I don't think we should say "reasons to scrape the targeted website", because scrapers can be actors, but not ALL actors are scrapers. Actors can be anything!

I'm really trying to avoid labelling actors as JUST for scraping, so maybe it would be better to say something like "common use cases" or something like that.